### PR TITLE
RC: Enabling private service connection causes issues with the public endpoint

### PR DESCRIPTION
**Pull request by cmilesb**
Ticket: DOC-3526
Staging link: https://docs.redis.com/staging/DOC-3526/rc/security/private-service-connect/

### CODE REVIEW

File: content/rc/security/private-service-connect.md [hunk 1]
```diff
@@ -132,6 +132,10 @@ Once your Private Service Connect endpoint is active, you can connect your appli
 
 1. Copy the endpoint and use it in your application to connect to your database.
 
+{{< note >}}
+Once your Private Service Connect endpoint is active, you must connect to the database from the Private Service Connect endpoint. If you previously used the public or private endpoint to connect to your database, you must migrate any connections to the Private Service Connect endpoint.
```
Comment (joniredis) [inline content/rc/security/private-service-connect.md:136]: @cmilesb do you think we should also mention that customers moving away from PSC should change their endpoint to the public or private endpoint or do you think the above implies it?

**Review by joniredis — APPROVED** [review]

File: content/rc/security/private-service-connect.md (hunk 1 above)
Comment (joniredis) [reply content/rc/security/private-service-connect.md:136]: Other than that looks good :) Thanks!

Comment (cmilesb) [reply content/rc/security/private-service-connect.md]: Just added something under deactivate in [eb06970](https://github.com/RedisLabs/redislabs-docs/pull/3195/commits/eb06970dcb6c3cd955e292dd558c4b1c5c1ef529)

**Review by dwdougherty — APPROVED** [review]
LGTM

**Pull request merged**